### PR TITLE
multiple improvements on exported types

### PR DESCRIPTION
**Pull request by GermanBluefox**

### CODE REVIEW

**Review by foxriver76 — APPROVED** [review]

File: packages/types-dev/objects.d.ts [hunk 1]
```diff
@@ -376,6 +376,9 @@ declare global {
 
             // Make it possible to narrow the object type using the custom property
             custom?: undefined;
+
+            /** disable sentry reporting */
+            disableDataReporting?: boolean;
```
Comment (foxriver76) [inline packages/types-dev/objects.d.ts:381]: We have a state for this to control plugins active state as already discussed, see also https://github.com/ioBroker/ioBroker.admin/issues/2409

**Review by foxriver76 — CHANGES_REQUESTED** [review]

File: packages/types-dev/objects.d.ts (hunk 1 above)
Comment (GermanBluefox) [reply packages/types-dev/objects.d.ts]: disableDataReporting removed

**Review by foxriver76 — APPROVED** [review]

**Comment by GermanBluefox** [comment]
@foxriver76 @Apollon77 what is with this PR?

**Comment by foxriver76** [comment]
CI is failing @GermanBluefox

**Comment by GermanBluefox** [comment]
> CI is failing @GermanBluefox
> 
> i would strongly recommend to put the ids back to simple strings for now

Done

**Pull request merged**